### PR TITLE
refactor: remove find command and hardcode source and header files in the makefile

### PR DESCRIPTION
**Pull request by gahtan-syarif**
successfully compiles
fix https://github.com/Disservin/fast-chess/issues/562

### CODE REVIEW

File: app/Makefile [hunk 1]
```diff
@@ -134,9 +233,59 @@ $(BUILDDIR)/%.o: %.cpp | build_directories
...
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test/add-subdirectory-test
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test/compile-error-test
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test/cuda-test
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test/find-package-test
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test/fuzzing
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test/gtest
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test/gtest/gmock
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test/gtest/gtest
+	@$(MKDIR) $(BUILDDIR)/third_party/fmt/test/static-export-test
```
Comment (Disservin) [inline app/Makefile:287]: you dont need this, ony "gzip" dir is required

File: app/Makefile [hunk 2]
```diff
@@ -134,9 +233,59 @@ $(BUILDDIR)/%.o: %.cpp | build_directories
 
 build_directories:
```
Comment (Disservin) [inline app/Makefile:234]: mh ill have to look if there is a better way to solve this

**Review by Disservin — CHANGES_REQUESTED** [review]

File: app/Makefile (hunk 1 above)
Comment (gahtan-syarif) [reply app/Makefile]: removed 👍

File: app/Makefile (hunk 2 above)
Comment (gahtan-syarif) [reply app/Makefile:234]: in the meantime i think its good to merge this first, so we can go ahead with the fishtest integration

**Pull request merged**